docs/README.md

```diff
@@ -56,10 +63,31 @@ The second annotation type is a **comment**. Comments are text annotations, just
 Comments can also be *Upvoted*, *Downvoted* or *Removed* by other users. 
 
 ### 3. GEOTAGGING
-Lastly, **geotagging** is a special kind of tag that refers specifically to a location. A user can start typing the location name in the input field under the map, and then select the appropriate location from the suggested ones on the drop-down below. Geotags can also be *Upvoted*, *Downvoted* or *Removed* by other users.
+**Geotagging** is a special kind of tag that refers specifically to a location. A user can start typing the location name in the input field under the map, and then select the appropriate location from the suggested ones on the drop-down below. Geotags can also be validated by the supported feedback methods (upvoting or percentage rating).
 
 <img src="https://raw.githubusercontent.com/ails-lab/CrowdHeritage_Documentation/main/docs/_media/image9.jpg" width="auto">
 
+### 4. COLORTAGGING
+
+**Colortagging** is a special kind of tag that refers to color. A user can select the color from a predefined color pallette. Colortags can also be validated by the supported validation methods (upvoting or percentage rating).
+
+<img src="_media/annotating-colortagging.png" width="auto">
+
+### 5. IMAGETAGGING
+
+**IMAGETAGGING** is a special kind of tag that refers to tagging images. A user can see the original image along with algorithm-generated ones (e.g. by Super-Resolution models). The user can then click *Compare* to start the comparison.
+
+<img src="_media/annotating-image-tagging.png" width="auto">
+
+The original image is displayed at the top of the screen and below it (from left to right) the first algorithm-generated image, the original image in the same size as the computer-generated ones and the second algorithm-generated image. The user can zoom in a smaller window of the original image at the top and automatically the images at the bottom are displaying this smaller window. If the user wants to see the full images again he can stop the zoom process.
```

Comment:
The rest regarding image comparison were copied from the respective doc. 

https://docs.google.com/document/d/1IXH2I27FlY5U0QIz_5ImdytMkiJCX6_5YzhsbPD5P94/edit